roles/network_plugin/calico/tasks/install.yml

```diff
@@ -441,3 +441,14 @@
   when:
     - peer_with_router|default(false)
     - inventory_hostname in groups['k8s_cluster']
+
+- name: Calico | Configure ipam strictaffinity to true
+  command:
+    cmd: "{{ bin_dir }}/calicoctl.sh ipam configure --strictaffinity=true"
```

Comment:
From what I can tell this approach will only enable strict affinity, if a deployer later decides to disable strict affinity this will not help. If so, why does this need to be part of the deployment process if it can be easily enabled via calicoctl later on? Ideally the deployment tool should configure parameters that end up in configuration files that are later more difficult to change.

Reply:
You are totally right @cristicalin I'm changing this so that it will `enable` or `disable` (defualt) the strictaffinity accordingly

Reply:
a26d80d